src/vring.rs

```diff
@@ -159,9 +163,16 @@ impl<M: GuestAddressSpace> VringState<M> {
 
     /// Set queue addresses for descriptor table, available ring and used ring.
     pub fn set_queue_info(&mut self, desc_table: u64, avail_ring: u64, used_ring: u64) {
-        self.queue.state.desc_table = GuestAddress(desc_table);
-        self.queue.state.avail_ring = GuestAddress(avail_ring);
-        self.queue.state.used_ring = GuestAddress(used_ring);
+        // TODO: the errors returned by the following calls should be handled.
```

Comment:
Ditto, though this time the error will reach the users through `vhost-user`.

Reply:
Done.